### DISCUSSION TITLE
primeng template- how to use angular material style for special component

### DISCUSSION THREAD

**Discussion by rahim99** [primefaces/community#653] [PrimeNG Templates] [answered]
I have a diamond template and I want to use angular material styles for special component (for example it could be p-dropdown)
Is that possible?

Or maybe it is possible to change all components styles to use angular material styles?

**Comment by cetincakiroglu** [comment, marked as answer]
Hi,

Unfortunately, it's not possible yet. However, we're working on passthrough and unstyled mode to handle such cases in the future. You can change the component by custom CSS but I'm not sure if it is worth the time.
